tests/Unit/Informer/CMakeLists.txt

```diff
@@ -13,3 +13,7 @@ add_test_library(
   "${LIBRARY_SOURCES}"
   "Informer;Utilities"
   )
+spectre_add_python_test(
+  "Unit.Informer.Python"
+  "Test_MakeTimeInfoFromBuild.py"
+  "unit;Informer;python")
```

Comment:
can you make the capitalization here consistent? We seem to be capitalizing the first letter up above (line 14)